intg/src/main/java/org/apache/atlas/model/lineage/AtlasLineageInfo.java

```diff
@@ -19,6 +19,7 @@
 
```

Comment:
add metric logs in lineageContainsEdgeV2

Reply:
addressed